R/derive_var_confirmation_flag.R

```diff
@@ -0,0 +1,350 @@
+#' Derive Confirmation Flag
+#'
+#' Derive a flag which depends on subsequent observations of the dataset. For
+#' example, flagging events which need to be confirmed by a second event.
+#'
+#' An example usage might be flagging if a patient received two required
+#' medications within a certain timeframe of each other.
+#'
+#' In the oncology setting, for example, the function could be used to flag if a
+#' response value can be confirmed by a subsequent assessment. This is commonly
+#' used in endpoints such as best overall response.
+#'
+#' @param dataset Input dataset
+#'
+#'   The variables specified by the `by_vars` and `join_vars` parameter are
+#'   expected.
+#'
+#' @param by_vars By variables
+#'
+#'   The specified variables are used as by variables for joining the input
+#'   dataset with itself.
+#'
+#' @param order Order
+#'
+#'   The observations are ordered by the specified order.
+#'
+#' @param new_var New variable
+#'
+#'   The specified variable is added to the input dataset.
+#'
+#' @param join_vars Variables to keep from joined dataset
+#'
+#'   The variables needed from the subsequent observations should be specified
```

Comment:
"subsequent" again - check and update any places this is used in this documentation or `filter_confirmation`